File: swagger/models/upcoming_autopay_event.py
from .base_model import BaseModel


class UpcomingAutopayEvent(BaseModel):
	"""
	no description available

	Attributes:

		client_contract_id
		The ID of the contract.

		charge_amount
		The amount charged.

		payment_method
		The payment method.

		schedule_date
		The date and time of the next payment.
	"""

	input_map = {
		'ClientContractId': 'client_contract_id',
		'ChargeAmount': 'charge_amount',
		'PaymentMethod': 'payment_method',
		'ScheduleDate': 'schedule_date',
		}

	output_map = {
		'client_contract_id': 'ClientContractId',
		'charge_amount': 'ChargeAmount',
		'payment_method': 'PaymentMethod',
		'schedule_date': 'ScheduleDate',
		}

	client_contract_id = None
	charge_amount = None
	payment_method = None
	schedule_date = None

File: swagger/Models/UpcomingAutopayEvent.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class UpcomingAutopayEvent extends BaseModel
{
	/**
	 * The ID of the contract.
	 * 
	 * @var integer format:int32
	 */
	public $clientContractId;
	/**
	 * The amount charged.
	 * 
	 * @var number format:double
	 */
	public $chargeAmount;
	/**
	 * The payment method.
	 * 
	 * @var string 
	 */
	public $paymentMethod;
	/**
	 * The date and time of the next payment.
	 * 
	 * @var string format:date-time
	 */
	public $scheduleDate;

	protected function getInputMap()
	{
		return [
			'ClientContractId' => ['clientContractId', 'integer', 'int32'],
			'ChargeAmount' => ['chargeAmount', 'number', 'double'],
			'PaymentMethod' => ['paymentMethod', 'string', null],
			'ScheduleDate' => ['scheduleDate', 'string', 'date-time'],
			];
	}

	protected function getOutputMap()
	{
		return [
			'clientContractId' => ['ClientContractId', 'integer', 'int32'],
			'chargeAmount' => ['ChargeAmount', 'number', 'double'],
			'paymentMethod' => ['PaymentMethod', 'string', null],
			'scheduleDate' => ['ScheduleDate', 'string', 'date-time'],
			];
	}
}
